gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionSha256Sum=9af5c8e7e2cd1a3b0f694a4ac262b9f38c75262e74a9e8b5101af302a6beadd7
```

Comment:
What's with the change here?

Reply:
Oh, that is accidental -- however I was meaning to ask the team if we should keep it.  Recent IntelliJ versions do NOT like having this present and will refuse to do gradle builds until you remove it.  I'll bring it up in a discussion and we can decide if it should stay here, otherwise I'll pull it out.